spec/controllers/idv/cancellations_controller_spec.rb

```diff
@@ -212,7 +212,7 @@
 
       context 'with in establishing in-person enrollment' do
         let(:user) { build(:user, :with_establishing_in_person_enrollment) }
-        let(:enrollment) { user.establishing_in_person_enrollment }
+        let!(:enrollment) { user.establishing_in_person_enrollment }
```

Comment:
This had me tearing my hair out because the enrollment stopped existing in tests here which seemed orthogonal to my changes.

This really should have been `let!` from the start. But the `if user.establishing_in_person_enrollment` check in ProofingComponents was causing this to pop into existence as a stealthy side effect. Removing that code as no longer necessary removed that unnoticed side effect and suddenly these tests failed because they didn't have an enrollment. 🫨